lib/services/thinkingParser.js

```diff
@@ -298,6 +364,12 @@ function blackButtonCloseRequest(fields, result, timestamp, callback) {
         metadatas: timestamp
     });
     result.attributes.push({
+        name: 'aux_op_status',
+        value: 'CLOSED',
```

Comment:
In fact, it is already documented as an issue at https://github.com/telefonicaid/iotagent-thinking-things/issues/50 :)